dvc/command/commit.py

```diff
@@ -65,6 +65,9 @@ def add_parser(subparsers, parent_parser):
         help="Commit cache for subdirectories of the specified directory.",
     )
     commit_parser.add_argument(
-        "targets", nargs="*", default=None, help="DVC-files."
```

Comment:
Notice that I removed the `default` property from here and from the same arg (`targets`) in data_sync.py, because `None` is the default value for `default` anyway (and no other call to `add_argument("targets", ...)` in the repo sends it).

However, argparse code for `cache dir`/`value`, `config`/`value`, data sync/`-j`, `diff`/`-t` and `b_ref`, `gc`/`-j` and `-p`, and `run`/`-c` and `-w` still explicitly send an unnecessary `default=None` (unless I'm wrong and it's not unnecessary). Should I remove it from all those as well? Thanks

Reply:
On this, I'll take the liberty to remove the `default='None'`s for a future PR. Thanks